server/handlers.go

```diff
@@ -0,0 +1,283 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/http"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/eth"
+)
+
+func logAndRespondWithError(w http.ResponseWriter, errMsg string, code int) {
+	glog.Error(errMsg)
+	http.Error(w, errMsg, code)
+}
+
+func mustHaveFormParams(h http.Handler, params ...string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if err := r.ParseForm(); err != nil {
+			glog.Error(err)
+			logAndRespondWithError(w, "parse form error", http.StatusInternalServerError)
```

Comment:
Perhaps we could have some more concise helpers, eg

```
func respondWith500(w, err) {
  logAndRespondWithError(w, err, http.StatusInternalServerError)
}
func respondWith400(w, err) {
  logAndRespondWithError(w, err, http.StatusBadRequest)
}
```
(The "logAnd" prefix doesn't really seem necessary for the helpers, but that's just my personal preference.)

Reply:
Updated [here](https://github.com/livepeer/go-livepeer/pull/658/files#diff-105fedf355a9fcad82982745cef24f7aR15)